skada/deep/tests/test_deep_adversarial.py

```diff
@@ -114,6 +114,55 @@ def test_cdan(domain_classifier, domain_criterion, num_feature, max_feature, n_c
     assert history[0]["train_loss"] > history[-1]["train_loss"]
 
 
+@pytest.mark.parametrize(
+    "domain_classifier, criterion, num_features",
+    [
+        (DomainClassifier(num_features=10), CrossEntropyLoss(), None),
+        (DomainClassifier(num_features=10), None, None),
+        (None, None, 10),
+    ],
+)
+def test_mdd(domain_classifier, criterion, num_features):
+    module = ToyModule2D()
+    module.eval()
+
+    n_samples = 20
+    dataset = make_shifted_datasets(
+        n_samples_source=n_samples,
+        n_samples_target=n_samples,
+        shift="concept_drift",
+        noise=0.1,
+        random_state=42,
+        return_dataset=True,
+    )
+
+    method = MDD(
+        ToyModule2D(),
```

Comment:
In your case, I assume, it should be `ToyModule2D(n_classes=5, proba=True),` or something like that but clearly not empty otherwise it'll have an output shape of (n, 2)